services/QuillLMS/app/queries/impact_metrics/schools_containing_certain_teachers_query.rb

```diff
@@ -29,7 +22,15 @@ def from_and_join_clauses
 
     def where_clause
       <<-SQL
-        WHERE schools_users.user_id IN (#{teacher_ids.join(', ')})
```

Comment:
This was the line that resulted in a query that was too large for BigQuery to accept